qiskit_ibm_runtime/utils/backend_converter.py

```diff
@@ -161,11 +160,11 @@ def convert_to_target(
                         if any(qubit in faulty_qubits for qubit in qarg):
                             continue
                         target[inst][qarg].calibration = sched
-        if "delay" not in target:
-            target.add_instruction(
-                Delay(Parameter("t")),
-                {(bit,): None for bit in range(target.num_qubits) if bit not in faulty_qubits},
-            )
+    if "delay" not in target:
```

Comment:
If `target` is still `None` here won't there be an error?

Reply:
`target` should be constructed in either case, whether `properties` are `None` or not. Do you see a case where `target` is still undefined? We may need to add a couple of tests here

Reply:
Ah I see, you're right I got confused with the identations - originally the indentation was added because `faulty_qubits` could be undefined.

This LGTM